event-processing/obfuscation/tests/unit/app.test.ts

```diff
@@ -60,15 +99,89 @@ describe('Unit test for app handler', function () {
         const result = await handler(firehoseEvent);
 
         expect(result).toEqual(expectedResult);
-        var resultData = Buffer.from(result.records[0].data, 'base64').toString('ascii');
     });
 
-    it('removes all additional fields if its a single event', async () => {
-        const expectedData: IObfuscatedEvent = {
-            timestamp: 1609462861,
-            timestamp_formatted: '2021-01-23T15:43:21.842',
-            event_name: 'AUTHENTICATION_ATTEMPT',
-        };
+    it('removes all additional fields if its a single event with secret string', async () => {
```

Comment:
Is this now 'removes all additional fields' or should it be 'obfuscate fields'

Reply:
I think this is because originally it was just removing the fields. So we can just update this.

Reply:
```suggestion
    it('obfuscates expected fields if its a single event using the secret string', async () => {
```